packages/core/src/app/address/googleAutocomplete/GoogleAutocomplete.spec.tsx

```diff
@@ -19,4 +19,16 @@ describe('GoogleAutocomplete Component', () => {
 
         expect(toJson(tree)).toMatchSnapshot();
     });
+
+    it('renders input with max length', () => {
```

Comment:
This seems like a small enough test file, can we convert the suite to RTL?

Reply:
@animesh1987 Converted it.